components/Footer/Nav.js

```diff
@@ -0,0 +1,30 @@
+import Image from "next/image";
+import Link from "next/link";
+import React from "react";
+import { StyledNav } from "./StyledNav";
+
+export default function Nav() {
+  return (
+    <StyledNav>
+      <Link href="/">
+        <Image
+          className="home"
+          src="/appstore-outlined.svg"
+          width={36}
+          height={36}
+          alt="home icon"
+        />
+      </Link>
+
+      <Link href="/favourites">
+        <Image
+          className="fav"
+          src="/favorite-add-outline.svg"
+          width={36}
+          height={36}
+          alt="add-fav icon"
```

Comment:
The alt attribute is for screenreaders and in case the image is not shown I think, therefore I would make it a little more descriptive and not shorten the word favorite :)

Reply:
Thanks! Changed it.